docs/spec/tempo.grafana.com_tempomonolithics.yaml

```diff
@@ -123,6 +123,7 @@ spec:                                    # TempoMonolithicSpec defines the desir
       azure:                             # Azure defines the configuration for Azure Storage.
         secret: ""                       # Secret is the name of a Secret containing credentials for accessing object storage. It needs to be in the same namespace as the TempoMonolithic custom resource.
       backend: "memory"                  # Backend defines the backend for storing traces. Default: memory.
+      credentialMode: ""                 # CredentialMode can be used to set the desired credential mode for authenticating with the object storage. If this is not set, then the operator tries to infer the credential mode from the provided secret and its own configuration.
```

Comment:
Can you move this setting under the `s3` object, as it is only supported for the `S3` backend?

Reply:
Sure.